extensions/amp-accordion/1.0/accordion.js

```diff
@@ -426,11 +489,13 @@ export function AccordionContent({
     <Comp
       {...rest}
       ref={ref}
-      className={`${className} ${classes.sectionChild} ${classes.content}`}
+      className={`${className} ${classes.sectionChild} ${classes.content} ${
+        !expanded ? hiddenClass : ''
+      }`}
       id={contentId}
       aria-labelledby={headerId}
       role={role}
-      hidden={!expanded}
+      aria-hidden={String(!expanded)}
```

Comment:
`aria-hidden` should not be necessary with `display: none` or `visibility: hidden`, so we should verify if it is needed for `content-visibility: hidden-matchable`. Played around with it and seems a bit buggy. cc @vmpstr as FYI

Reply:
If it's buggy for `content-visibility: hidden-matchable` would it make sense to keep it to be safe?

Reply:
Let's confirm with the spec folk. If intention is to make it implied with `hidden-matchable`, then, yes, I'm in favor of removing.

Reply:
Removed for now.  Can add back in if we hear differently from @vmpstr.  Thanks!!

Reply:
Confirmed with @caroqliu offline who has been in contact with @vmpstr that we do not need `aria-hidden` when `content-visibility: hidden-matchable` is applied to the element and that the observed buggy behavior is currently in progress to be fixed and should be in Canary shortly.